plasmapy/dispersion/numerical/kinetic_alfven_.py

```diff
@@ -245,67 +215,68 @@ def kinetic_alfven(  # noqa: C901, PLR0912, PLR0915
     elif np.isscalar(theta):
         theta = np.array([theta])
 
-    n_e = z_mean * n_i
+    Z = ion.charge_number
+    n_e = Z * n_i
     c_s = speed.ion_sound_speed(
         T_e=T_e,
         T_i=T_i,
         ion=ion,
         n_e=n_e,
         gamma_e=gamma_e,
         gamma_i=gamma_i,
-        Z=z_mean,
+        Z=Z,
     )
-    v_A = speed.Alfven_speed(B, n_i, ion=ion, z_mean=z_mean)
-    omega_ci = pfp.gyrofrequency(B=B, particle=ion, signed=False, Z=z_mean)
+    v_A = speed.Alfven_speed(B=B, density=n_i, ion=ion)
+    omega_ci = pfp.gyrofrequency(B=B, particle=ion, signed=False)
 
     # parameters kz
     omega = {}
-    for Theta in theta:  # TODO: should be pretty simple to vectorize this, no?
-        kz = np.cos(Theta) * k
+    for θ in theta:  # vectorize this?
```

Comment:
Can you use any unicode character in python scripts? Looks a lot better with the theta symbol